### PR TITLE
feat(Inji-410) - Add support for eldoria theme

### PR DESCRIPTION
**Pull request by PuBHARGAVI**

### CODE REVIEW

File: android/app/build.gradle [hunk 1]
```diff
@@ -164,7 +164,10 @@ android {
 
         manifestPlaceholders = [
             APP_NAME: APP_NAME_RELEASE,
-            GOOGLE_NEARBY_MESSAGES_API_KEY: "${properties.getProperty('GOOGLE_NEARBY_MESSAGES_API_KEY')}"
+            GOOGLE_NEARBY_MESSAGES_API_KEY: "${properties.getProperty('GOOGLE_NEARBY_MESSAGES_API_KEY')}",
+            app_name_eldoria: "Eldoria",
+            app_name_inji: "Inji",
+            app_name_veridonia: "Veridonia",
```
Comment (swatigoel) [inline android/app/build.gradle:170]: Do we need all these names?
or only app_name: "Eldoria"

File: screens/QrLogin/QrLoginSuccessMessage.tsx [hunk 2]
```diff
@@ -7,10 +7,17 @@ import { Centered, Button, Text, Column } from '../../components/ui';
...
 import { QrLoginRef } from '../../machines/QrLoginMachine';
+import { APPLICATION_THEME } from 'react-native-dotenv';
 
 export const QrLoginSuccess: React.FC<QrLoginSuccessProps> = (props) => {
   const { t } = useTranslation('QrScreen');
   const controller = useQrLogin(props);
+  const clientName: Map<string, any> = new Map<string, any>([
+    ['purple', controller.clientName],
+    ['default', controller.clientName],
+    ['eldoria', 'Eldoria Authentication Platform'],
```
Comment (swatigoel) [inline screens/QrLogin/QrLoginSuccessMessage.tsx:18]: Should we rely on clientName coming from esignet only?

Comment (PuBHARGAVI) [reply screens/QrLogin/QrLoginSuccessMessage.tsx:18]: Are we getting clientName from the esignet?

File: android/app/build.gradle (hunk 1 above)
Comment (sree96) [reply android/app/build.gradle:170]: This is not used. will remove it

**Review by swatigoel — APPROVED** [review]

**Pull request merged**